libs/service-portal/health/src/screens/HealthOverview/HealthOverview.tsx

```diff
@@ -34,6 +36,23 @@ export const HealthOverview = () => {
 
   const intl = useIntl()
 
+  const featureFlagClient = useFeatureFlagClient()
+
+  const [enabledPaymentPage, setEnabledPaymentPage] = useState<boolean>(false)
+
+  useEffect(() => {
+    const isFlagEnabled = async () => {
+      const ffEnabled = await featureFlagClient.getValue(
+        `isLicenseServiceDrivingLicenceClientV2Enabled`,
+        false,
+      )
+      if (ffEnabled) {
+        setEnabledPaymentPage(ffEnabled as boolean)
```

Comment:
```suggestion
        setEnabledPaymentPage(true)
```

since we now it's true, or just pass it without `as`.